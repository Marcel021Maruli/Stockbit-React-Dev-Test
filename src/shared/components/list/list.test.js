import React from 'react';
import List from './list';
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const mockList = { Title: "Batman", Poster: "someUrl", Year: "2021", Type: "movie", imdbID: 'some ID' };

describe("<List />", () => {
  const wrapper = shallow(<List {...mockList} />)

  it('should renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})