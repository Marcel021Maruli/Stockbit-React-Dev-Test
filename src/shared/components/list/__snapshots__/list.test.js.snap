// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should renders correctly 1`] = `ShallowWrapper {}`;
